config/ams.yml

```
@@ -0,0 +1,43 @@
+
+test:
+  <<: *default
+  base_url: "https://ams.example.com"
+  subscription_prefix: "test"
+  subscriptions_path: "/v1/projects/test/subscriptions"
+  topics_path: "/v1/projects/test/topics"
+  token: "test-token"
+  topics:
+    - service-create
+    - service-update
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)